pkg/k8s/node.go

```diff
@@ -230,15 +246,67 @@ func setNodeNetworkUnavailableFalse(c kubernetes.Interface, nodeName string) err
 	return err
 }
 
+// removeNodeTaint removes the AgentNotReadyNodeTaint from nodes' taints which
```

Comment:
comment wording nit: 

"removeNodeTaint removes the AgentNotReadyNodeTaint allowing for pods to be scheduled...
...
"